xbmc/cores/VideoPlayer/DVDCodecs/Video/VAAPI.cpp

```diff
@@ -2784,11 +2784,6 @@ bool CFFmpegPostproc::PreInit(CVaapiConfig &config, SDiMethods *methods)
 {
   m_config = config;
   bool use_filter = true;
-  if (!m_dllSSE4.Load())
```

Comment:
why?

Reply:
As the commit message says: This function will always return false on mesa, in which case there is no point in loading `m_dllSSE4`. So I moved the loading further down when we know whether we actually need it or not.

Reply:
ok